addons/remove-curved-stage-border/addon.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Remove Curved Stage Border",
+  "description": "Removes the curved border around the stage, allowing you to see the very corners.",
```

Comment:
```suggestion
  "description": "Removes the curved border around the stage, allowing you to see into the corners.",
```

Reply:
"the very corners" is valid. What did you see in the corner? We want to see the right-angle corners, not something inside of them.

Reply:
"the very corners" just sounds odd imo

Reply:
If you think it is, then "the corners" is enough, "into the corners" would be weirder/misleading. Otherwise, it is fine.